src/base/rss/rss_autodownloader.cpp

```diff
@@ -245,6 +245,7 @@ void AutoDownloader::processJob(const QSharedPointer<ProcessingJob> &job)
         }
 
         rule.setLastMatch(job->articleDate);
+        rule.matched(job->articleTitle);
```

Comment:
>I've just been trying to implement this, but I came across the fact the matches method is const, and can't have side effects. That makes saving the computed episode name impossible.

If you don't pay attention, I offered to create `mutable` field which can be modified by const methods.
Having something like AutoDownloadRuleMatch is another way.